### DISCUSSION TITLE
Re-orderable tables

### DISCUSSION THREAD

**Discussion by oddvalue** [filamentphp/filament#3153] [Ideas]
If you have a model that's ordered by a sorting column and want to update the order of those records from the table the only way suggested anywhere is this trick on the Filament website:
https://filamentphp.com/tricks/reorder-table-records

It would be good to have the ability to define a table as re-orderable and then use the livewire sortable plugin already in use on the repeater fields to support ordering them.

I'm happy to submit a PR if this is a feature is something that people agree would be at home on the tables package.

**Comment by danharrin** [comment]
It's on our roadmap already, but we need to be careful as it will conflict with sorting, pagination, searching and filtering

**Reply by oddvalue** [reply]
I think for the feature to work you would need to disable pagination when a resource is reorderable and disable reordering when the sorting has been changed from the default sort column or any search/filter is applied. 

I have seen data tables that support reordering over paginated data by allowing you to drop an item on to the previous or next page by dragging them to the top or bottom of the list respectively. That said, I'm not a fan of that particular UX.

**Reply by danharrin** [reply]
Added to v2 in #3211
